src/js/view/controls/controlbar.js

```diff
@@ -442,6 +473,13 @@ export default class Controlbar {
     onFullscreen(model, val) {
         utils.toggleClass(this.elements.fullscreen.element(), 'jw-off', val);
     }
+              
+    checkDvrLiveEdge() {
```

Comment:
@johnBartos it always returns false. The original seek to live implementation https://github.com/jwplayer/jwplayer/pull/2205/files#diff-da6bb0847c43fc9abf65d5ce7b9829c5R356 is consistent with my check